src/app.js

```diff
@@ -1,61 +1,102 @@
 import './style.css';
-import { setCompleted } from './function.js';
+import {
+  createlocalstorage, addBtn, listArr,
+} from './script.js';
 
 const container = document.querySelector('.list-container');
+const deleteallbtn = document.querySelector('.clear');
 
-let todoList = [
-  {
-    index: 0,
-    description: 'Wash the dishes',
-    completed: true,
-  },
-
-  {
-    index: 1,
-    description: 'Complete todo lis project',
-    completed: true,
-  },
-
-  {
-    index: 2,
-    description: 'Play game with my girlfriend',
-    completed: true,
-  },
-
-  {
-    index: 3,
-    description: 'Eat some cookies',
-    completed: true,
-  },
-];
-
-const storedData = JSON.parse(localStorage.getItem('todos'));
-if (storedData) {
-  todoList = storedData;
-}
-
-const display = () => {
-  todoList.forEach((item) => {
-    const containerItems = `
-    <div class="task">
-    <div class="wrapper">
-        <input type="checkbox" name="" id="boxes" class="box m-input" data-id=${item.index} ${item.completed ? 'checked' : ''}>
-        <input type="text" name="" data-id="${item.index}" value="${item.description}">
-    </div>
-        <i id="ellipse" class="fas fa-ellipsis-v m-i"></i>
-        
-    </div>
-    <hr>`;
-    container.innerHTML += containerItems;
+const display = (array) => {
+  let containerItems = '';
+  array.forEach((item) => {
+    containerItems += `
+      <div class="task">
+      <div class="wrapper">
+          <input type="checkbox" name="" id="boxes" class="box m-input" data-id=${item.index} ${item.completed ? 'checked' : ''}>
+          <input type="text" name="" data-id="${item.index}" id="${item.index}" class="input-text" value="${item.description}">
+      </div>
+          <i id="ellipse" class="fas fa-ellipsis-v m-i"></i>
+         <i id="deletebtn" class="fas fa-trash"></i>
+          
+      </div>
+      <hr>`;
   });
 
+  container.innerHTML += containerItems;
+
   const boxes = document.querySelectorAll('.box');
 
   boxes.forEach((box) => {
     box.addEventListener('change', () => {
-      setCompleted(todoList, box);
+      const inputtext = document.getElementById(box.dataset.id);
+
+      const founditem = listArr.find((item) => box.dataset.id === item.index.toString());
+      founditem.completed = box.checked;
+      if (founditem.completed === true) {
+        inputtext.style.textDecoration = 'line-through';
+      } else {
+        inputtext.style.textDecoration = 'none';
+      }
+      localStorage.setItem('todos', JSON.stringify(listArr));
     });
   });
 };
 
-display();
+addBtn.addEventListener('click', () => {
+  createlocalstorage(listArr);
+  window.location.reload();
+});
+display(listArr);
```

Comment:
- [ ] Please implement a button with an event listener as the previous reviewer requested. Currently, it is not implemented yet. This can come because you have some uncommitted code. If that is the case please commit and ask for another review

Reply:
Please can you check the completedstatus.js file, I declared the function in there and imported it to app.js file
Then I called the createlocalstorage  function in line 46 of app.js file